tests/testthat/test-wikidata.R

```diff
@@ -42,6 +42,20 @@ test_that("wd_ident returns correct results", {
                            'drugbank', 'zvg', 'chebi', 'chembl', 'unii', 'source_url', 'query'))
```

Comment:
Can you also edit this test to include the new columns?  You might also consider combining your new test into this one in some way so there's less redundant testing.

```suggestion
                           'drugbank', 'zvg', 'chebi', 'chembl', 'unii', 'lipidmaps', 'swisslipids', 'source_url', 'query'))
```

Reply:
I will combine them.